rbac/management/group/definer.py

```diff
@@ -30,7 +30,7 @@
 def seed_group(tenant):
     """For a tenant create or update default group."""
     with tenant_context(tenant):
-        name = 'Platform group'
+        name = 'Default user access'
```

Comment:
I am afraid this might not update the current group name. We just get the platform_default=True group and update the roles in it below.

Reply:
Ahh, that's a good point. Let me fix that.

Reply:
Thinking more on it, would I need to put something in the code to handle existing group names, or just get a new migration created to affect the existing groups?

Reply:
I think we'd probably want a data migration to update the existing data as a one-off vs code to handle this. Moving forward, since the group name has a unique validation and disallow changing the name, we wouldn't need to worry about the names of these default groups being incorrect. Thoughts?

Reply:
That kinda fits along with my thinking, yeah.

Reply:
Agree 👍